client/app/components/packages/pas-form/edit.hbs

```diff
@@ -1,97 +1,103 @@
-
-      <Packages::PasForm::ProposedDevelopmentSite @form={{saveableForm}} />
-
-      <Packages::PasForm::ProjectDescription @form={{saveableForm}} />
-
-      <Packages::PasForm::AttachedDocuments
-        @form={{saveableForm}}
-        @model={{this.pasForm}}
-      />
-
-    </div>{{! end left/main column }}
-    <div class="cell large-4 sticky-sidebar">
-
-      <saveableForm.PageNav>
-        <saveableForm.SaveButton
-          @isEnabled={{or @package.isDirty saveableForm.isSaveable}}
-          @onClick={{this.savePackage}}
-          data-test-save-button
+  <saveablePackageForm.SaveableForm
+    @model={{saveablePackageForm.data.pasForm}}
+    @validators={{array this.validations.SaveablePasFormValidations this.validations.SubmittablePasFormValidations}}
+    as |saveablePasForm|
+  >
+    <div class="grid-x grid-margin-x">
+      <div class="cell large-8">
+
+        <section class="form-section">
+          <h1 class="header-large">
+            Pre-Application Statement
+            <small class="text-weight-normal">
+              {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
+            </small>
+          </h1>
+
+          <p>
+            The <strong>Pre-Application Statement</strong> requests pertinent information about a Site and Proposed Project. By
+            submitting the PAS, a prospective Applicant acknowledges that they intend to file a Land Use Application with NYC
+            Planning. Submission allows staff to be assigned from appropriate divisions and project tracking to commence.
+          </p>
+
+          <p>
+            NYC Planning understands that some projects may not be fully formed at the submission of the PAS. Give as much
+            detailed information as possible. Failure to provide detailed, relevant information will result in the PAS being
+            rejected.
+          </p>
+        </section>
+
+        <Packages::PasForm::ProjectInformation @form={{saveablePasForm}} />
+
+        <Packages::PasForm::ApplicantTeam
+          @form={{saveablePasForm}}
+          @addApplicant={{this.addApplicant}}
+          @removeApplicant={{this.removeApplicant}}
+          @validations={{this.validations}}
         />
 
-        <Packages::PasForm::PasFormError @package={{@package}} />
+        <Packages::PasForm::ProjectGeography @form={{saveablePasForm}} />
 
-        <saveableForm.SubmitButton
-          @isEnabled={{saveableForm.isSubmittable}}
-          class="secondary"
-          data-test-submit-button
+        <Packages::PasForm::ProposedLandUseActions @form={{saveablePasForm}} />
+
+        <Packages::PasForm::ProjectArea @form={{saveablePasForm}} />
+
+        <Packages::PasForm::ProposedDevelopmentSite @form={{saveablePasForm}} />
+
+        <Packages::PasForm::ProjectDescription @form={{saveablePasForm}} />
+
+        <Packages::PasForm::AttachedDocuments
+          @form={{saveablePasForm}}
```

Comment:
should this be `@form={{saveablePackageForm}}`?

Reply:
Weird, let me see

Reply:
Oh, you changed it in a later commit, i see

Reply:
ooh